base/set.jl

```diff
@@ -191,6 +191,24 @@ function unique(f::Callable, C)
 end
 
 """
+	unique!(itr)
+
+Removes recurrences of items and returns the modified collection
```

Comment:
Use imperative "Remove" and add ending dot. Also, better say "duplicated items" rather than "recurrence". Finally, would be worth saying a word about the fact that `isequal` is used (like for `unique`), and adding one or two examples (see how to do so in the Documentation section of the manual).